pmd-ui/src/main/java/net/sourceforge/pmd/util/fxdesigner/EventLogController.java

```diff
@@ -32,6 +37,8 @@
 
     private final DesignerRoot designerRoot;
 
+    private static final Duration PARSEEXCEPTION_DELAY = Duration.ofMillis(100);
```

Comment:
You should add an underscore between "parse" and "exception". Btw 100 millis is way too low, prop that up to 3000 and observe the change